conf/messages.en

```diff
@@ -729,6 +729,7 @@ partnership.removePartner.caption = Partners in the partnership
 amend.organisation.title = Your organisation’s details
 amend.individual.title = Your details
 amend.group.title = Representative member’s details
+amend.partnership.changeGroupLead.link = Change the representative member of the group
```

Comment:
can this be `amend.group.changeGroupLead.link` please

Reply:
```suggestion
amend. group.changeGroupLead.link = Change the representative member of the group
```